server/config/config.go

```diff
@@ -131,6 +131,8 @@ type Config struct {
 	logProps *log.ZapProperties
 
 	EnableConfigManager bool
+
+	EnableDashboard bool
```

Comment:
We need to always launch the dashboard handler.
After that, we will control the dashboard through dynamic configuration.

Reply:
Do you mean that the handler will be always online and add a dynamic check at entrance to reject or pass the request dynamically according to the configuration?